tests/integration/view-change.test.js

```diff
@@ -116,8 +133,9 @@ describe('view change', function () {
             if (ctx.numOfKeys > 0) {
 
                 it('should be able to fetch full keys list', async function () {
-                    await pTimeout(this.api.keys(), harnessConfigs.clientOperationTimeout, `Keys() failed to respond in ${harnessConfigs.clientOperationTimeout}`)
+                    this.api.keys()
                         .then(val => val.should.have.lengthOf(ctx.numOfKeys + 1))
+                        .catch(err => err)
```

Comment:
Please forgive my lack of JS expertise. IIRC the behaviour depends on the test framework/runner here. The catch/return of `err` still allows this test to fail, correct?

Reply:
Fixed